### PR TITLE
Fixed example to properly show col-half, col-quarter etc.

### PR DESCRIPTION
**Pull request by burakcan**
Fixes: https://github.com/IonicaBizau/gridly/issues/10

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -35,286 +35,760 @@ <h1 class="title">Gridly</h1>
...
+                    </div>
+                    <div class="col col-quarter">
+                        <div class="content">1 / 4</div>
+                    </div>
+                    <div class="col">
+                        <div class="content">9 / 20</div>
+                    </div>
                 </div>
                 <div class="row">
                     <div class="col">Two</div>
```
Comment (IonicaBizau) [inline example/index.html]: Is there a specific reason why you skipped this or you just missed it? :thought_balloon:

**Comment by IonicaBizau** [comment]
@burakcan Awesome work! :sparkles: :clap: 
Don't forget to add yourself as contributor in `package.json`. :grin:

File: example/index.html (hunk 1 above)
Comment (burakcan) [inline example/index.html]: seems like i've missed it :) fixing

Comment (burakcan) [inline example/index.html]: done :+1:

**Pull request merged**